docs/usage.rst

```diff
@@ -162,6 +162,46 @@ mapping that is checked against the :ref:`schema <schema_dict-rule>` rule:
    ``allow_unknown`` can also be set to a validation schema.
 
 
+.. _requiring-all:
+
+Requiring all
+-------------
+See also :ref:`this paragraph <require_all>`, :ref:`this paragraph <required>`.
+
+By default any keys defined in the schema are not required.
+However, you can require all document keys pairs by setting
+``require_all`` to ``True`` at validator initialization (``v = Validator(…, require_all=True)``)
+or change it latter via attribute access (``v.require_all = True``).
+``require_all`` can also be set as rule to configure a validator for a subdocument
```

Comment:
when `require_all` (the rule) is linked here.

Reply:
don't get how this should be changed.
```
``require_all`` can also be set as rule to configure a validator for a subdocument
that is checked against the :ref:`schema <schema_dict-rule>` rule:
```
→
```
``require_all`` can also be set as rule to configure a validator for a subdocument
when ``require_all`` (the rule) is linked here.
```
?

Reply:
i mean this:

```
``require_all`` can also be set :ref:`as rule <require_all>` to configure …
```